src/style.css

```
@@ -1,3 +1,99 @@
+:root{
+  --color: #00acdc;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: normal;
```

Default value, so you can just remove it
```suggestion
  font-weight: 500;
```